fineract-provider/src/main/java/org/apache/fineract/portfolio/group/api/CentersApiResource.java

```diff
@@ -155,6 +156,8 @@ public String retrieveAll(@Context final UriInfo uriInfo, @QueryParam("sqlSearch
         final ApiRequestJsonSerializationSettings settings = this.apiRequestParameterHelper.process(uriInfo.getQueryParameters());
         if (meetingDateParam != null && officeId != null) {
             Date meetingDate = meetingDateParam.getDate("meetingDate", dateFormat, locale);
+			LocalDate date = new LocalDate(meetingDate);
+			CenterReadPlatformServiceImpl.datePassed = date;
```

Comment:
Why are you assigning meeting date to a static variable in CenterReadPlatformServiceImpl